app/views/admin/instances/_instance_note.html.haml

```diff
@@ -0,0 +1,18 @@
+.report-notes__item
```

Comment:
This should be deleted.. we reuse the `report_notes/report_notes` partial — we should probably rename that partial to `moderation_notes` or something, since it applies for Account Moderation Notes, Report Notes, and Instance Notes now.